src/runtime/tests/schema-test.ts

```diff
@@ -515,4 +515,120 @@ describe('schema', () => {
     assert.strictEqual(getHash('nestedRefs'), 'Foo/num:Number|ref&[Bar/inner&[/val:Boolean|]str:Text|]');
     assert.strictEqual(getHash('refCollection'), '/rc@[Wiz/str:Text|]z:Number|');
   });
+  it('tests univariate schema level refinements are propogated to field level', async () => {
+    const manifest = await Manifest.parse(`
+      particle Foo
+        schema1: reads X {a: Number [a < 0], b: Number, c: Number} [a < 10]
+        schema2: reads Y {a: Number [a < 0 and a < 10], b: Number, c: Number}
+    `);
+    const schema1 = deleteLocations(manifest.particles[0].handleConnectionMap.get('schema1').type.getEntitySchema());
```

Comment:
Consider a helper function (local to this file) for extracting `manifest.particles[0].handleConnectionMap.get('${name}').type.getEntitySchema()`.